features/install.feature

```diff
@@ -5,7 +5,45 @@ Feature: Installation
   So that I can start testing without repetitive setup work.
 
 
-  Scenario: fresh install
+  Scenario: install with a .js test helper
+    Given I am working on a Node.js project
+    When I run "mycha install" and enter "spec/spec_helper.js"
```

Comment:
Yes, we can changes this separately. I guess the path is just there the spec_helper file is to be created then.